pulp_container/app/viewsets.py

```diff
@@ -482,3 +483,19 @@ class ContentRedirectContentGuardViewSet(ContentGuardViewSet):
     endpoint_name = "content_redirect"
     queryset = models.ContentRedirectContentGuard.objects.all()
     serializer_class = serializers.ContentRedirectContentGuardSerializer
+
+
+class ContainerNamespaceViewSet(
+    NamedModelViewSet,
+    mixins.CreateModelMixin,
+    mixins.DestroyModelMixin,
+    mixins.ListModelMixin,
+    mixins.RetrieveModelMixin,
+):
+    """
+    ViewSet for ContainerNamespaces.
+    """
+
+    endpoint_name = "pulp_container/namespaces"
```

Comment:
the changes look good to me, @dkliban do you want to take a look? do you have a different suggestion on the endpoint or we leave this one as is?

Reply:
i looked at 'django-admin show_urls' and i would not be opposed also having /namespaces/container/ as another option to consider